app/controllers/dishes_controller.rb

```diff
@@ -1,7 +1,14 @@
 class DishesController < ApplicationController
+  before_action :authenticate_request!
```

Comment:
Will there be any routes besides the login routes that won't need to be authenticated? Can this be moved to the ApplicationController?